public/config/configuration.json

```diff
@@ -8,6 +8,6 @@
   "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "KEYCLOAK_CONFIG_PATH": "/local-keycloak-config-url/keycloak.json",
   "ADDRESS_COMPLETE_KEY": "",
-  "LD_CLIENT_ID": "",
+  "LD_CLIENT_ID": "5f4d47179db2f40a33e88d32",
```

Comment:
By setting this, then locally we will have the same feature flag config as Dev, so we probably won't have to keep updating the defaults :P

OTOH, it's generally not a good idea to check in codes like this, since the repo is public. But I can't think of any security issue in this case. This key is only used to fetch a list of flags (and some logging). Thoughts?

Reply:
yes please!

Reply:
Yea, i think this is ok. 
If someone wants to use our flags so be it 😆